app/assets/javascripts/mobile/mobile_file_uploader.js

```diff
@@ -11,7 +11,7 @@ function createUploader(){
        action: "/photos",
        debug: false,
        button: document.getElementById('file-upload-publisher'),
-       sizeLimit: 4194304,
+       sizeLimit: gon.appConfig.settings.max_upload_size,
```

Comment:
I kept the same name in JS than in the diaspora.yml file here, should I change that?